snap/local/start-mysqld-exporter.sh

```diff
@@ -43,6 +44,6 @@ else
         --reuid snap_daemon \
         --regid snap_daemon \
         -- \
-        env DATA_SOURCE_NAME="${EXPORTER_USER}:${EXPORTER_PASS}@unix(${SOCKET})/" \
-        "$EXPORTER_PATH" "${EXPORTER_OPTS[@]}"
+        env MYSQLD_EXPORTER_PASSWORD="${EXPORTER_PASS}" \
+        "$EXPORTER_PATH" "--mysqld.username=${EXPORTER_USER}" "--mysqld.address=unix://${SOCKET}" "${EXPORTER_OPTS[@]}"
```

Comment:
```suggestion
        env MYSQLD_EXPORTER_PASSWORD="${EXPORTER_PASS}" \
            "${EXPORTER_PATH}" \
            "--mysqld.username=${EXPORTER_USER}" \
            "--mysqld.address=unix://${SOCKET}" \
            "${EXPORTER_OPTS[@]}"
```

Reply:
addressed in fa4e9a8c4b064b02274bd4bfa528966a8e690469